test/evergreen.yml

```diff
@@ -732,6 +740,46 @@ functions:
         permissions: public-read
         content_type: text/html
         remote_file: wiredtiger/${build_variant}/${revision}/${task_name}-${build_id}-${execution}/
+    
+  "aggregate-perf-stats":
+    - command: shell.exec
+      params:
+        shell: bash
+        script: |
+          set -o errexit
+          set -o verbose
+          # Collect all perf stats into a single file
+          python3 wiredtiger/bench/perf_run_py/aggregate_perf_stat.py
+    # Push the aggregated results to the 'Files' tab of the task in Evergreen
+    - command: s3.put
+      params:
+        aws_secret: ${aws_secret}
+        aws_key: ${aws_key}
+        local_files_include_filter: all_stats.csv
+        bucket: build_external
+        permissions: public-read
+        content_type: text/html
+        remote_file: wiredtiger/${build_variant}/${revision}/${task_name}-${build_id}-${execution}/  
+
+  "aggregate-perf-stats":
```

Comment:
Looks like git messed up and added this task twice during the rebase. Can you remove?